basic/README.md

```diff
@@ -39,8 +38,7 @@ A Microserivce using Spring Cloud Gateway to forward requests to consumer.
 
 * Testing
 
-Open in browser： http://localhost:9090/sayHello?name=World
-
+Open in browser： http://localhost:9090/sayHello?name=World or http://localhost:9090/sayHellofeign?name=zhangsan
```

Comment:
Open in browser： http://localhost:9090/sayHello?name=World or http://localhost:9090/sayHellofeign?name=World